src/main.rs

```diff
@@ -790,7 +790,11 @@ fn command_start< 'a >( matches: &clap::ArgMatches< 'a >, project: &CargoProject
         _ => None
     };
 
-    let server = rouille::Server::new( "localhost:8000", move |request| {
+
+    let port = std::env::var("PORT").unwrap_or("8000".into());
+    let host = std::env::var("HOST").unwrap_or("localhost".into());
```

Comment:
Looks good to me although I'd prefer less generic names, since environment variables are not namespaced.

Could you perhaps change `HOST` to `CARGO_WEB_HOST` and `PORT` to `CARGO_WEB_PORT`?

On a side note: if you'd like to pass the hostname and the port through the command line `--host` and `--port` command line arguments would also be fine to me. (Although I'm ok with the environment variables too provided they'll be prefixed to make sure they won't collide.)